core/trino-main/src/main/java/io/trino/operator/InterpretedHashGenerator.java

```diff
@@ -86,17 +85,6 @@ public long hashPosition(int position, Page page)
         return result;
     }
 
-    public long hashPosition(int position, IntFunction<Block> blockProvider)
```

Comment:
The inline comment in the other method should be removed too since it refers to this method.